upstream-sha256: fd8e0c382227432ca6489fb1a17ca18a2d5be84c137cec3a4af86a1dbfdafebb
actual-sha256: e4a9065d081241133bf689685af3a537582191095cf1a478f38ea921fea038cc
--- upstream.expected
+++ tolk.actual
@@ -17,5 +17,5 @@
   float val8 = (*(data1_100+(alu0+80)));
   float val9 = (*(data1_100+(alu0+90)));
   float val10 = (*(data2_10+alu0));
-  *(data0_10+alu0) = (val0+val1+val2+val3+val4+val5+val6+val7+val8+val9+val10);
+  *(data0_10+alu0) = (0.0f+val0+val1+val2+val3+val4+val5+val6+val7+val8+val9+val10);
 }
